test/sql/triggers.sql

```diff
@@ -324,4 +324,6 @@ CREATE TRIGGER t3 AFTER UPDATE ON transition_test REFERENCING NEW TABLE AS new_t
 CREATE TRIGGER t4 AFTER DELETE ON transition_test REFERENCING OLD TABLE AS old_trans FOR EACH ROW EXECUTE FUNCTION test_trigger();
 \set ON_ERROR_STOP 1
 
+-- Test insert blocker trigger does not crash when called directly
+SELECT _timescaledb_functions.insert_blocker();
```

Comment:
Would be good to set ON_ERROR_STOP before and after!

Reply:
Thanks, changed.